active/003-seq-feature.md

```diff
@@ -126,12 +134,19 @@ Another alternative is to make `BoundFeature` objects that are tightly coupled w
 This will ease the process of updating and deleting interval/features from the `IntervalMetadata` object.
 
 ## `BoundFeature` object structure
-This object is a mutable object, that contains attributes (i.e. `name`, `function`, `strand`, ...) as well as a list of intervals.  This object would have a [weakref](https://docs.python.org/3/library/weakref.html) to the corresponding interval object in the `IntervalTree`.  So if the intervals within `BoundFeature` are updated, the intervals within the `IntervalTree` are updated.
+This object is a *mutable* object, that contains attributes (i.e. `name`, `function`, `strand`, ...) as well as a list of intervals.  This object would have a [weakref](https://docs.python.org/3/library/weakref.html) to the corresponding interval object in the `IntervalTree`.  if the intervals within `BoundFeature` are updated, the intervals within the `IntervalTree` are updated. We will probably want to have a reserved keyword for `intervals` in the `BoundFeature` object.  For instance, if we wanted to grab all of the intervals associated with a features, we should be able to run something as follows
```

Comment:
I have no strong opinion, but just to probe this edge-case:

If I have a reference to a `BoundFeature` and allow the corresponding `Sequence` object to get GC'ed then I would have a semi-functional `BoundFeature` or would things stop working right afterwards?